source/administration/config-settings.rst

```diff
@@ -2683,6 +2699,28 @@ Lists installed plugins on your Mattermost server and whether they are enabled.
 | This feature's ``config.json`` setting is ``"PluginStates": {}`` with object input mapping plugin IDs as keys to objects, each of which contains a key ``"Enable": false`` with options ``true`` or ``false``. |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Require Plugin Signature
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**:
+
+**False**:
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"RequirePluginSignature": {}`` with object input mapping plugin IDs as keys to objects, each of which contains a key ``"Enable": false`` with options ``true`` or ``false``. |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Signature Public Key Files
```

Comment:
This is actually an array field, whose contents are relative or absolute paths to signature files. In addition to the Mattermost plugin signing key built-into the server, each public key specified here is trusted to validate plugin signatures.